Third person camera

notleeandrew opened the pull request:
This is a mouse controlled third person camera

notleeandrew committed:
Add files via upload

This is a mouse controlled third person camera

JammieDodgers commented:
Well executed and simple.

Works very well

Collision detection on the camera so it doesn't clip through objects

notleeandrew committed:
Add files via upload

this code will allow you to make the player move alongside making it make the player a child of the moving platform whilst colliding with the moving platform

notleeandrew committed:
Add files via upload

The kill player script will not work alone without the respawnplayer script as you must have a reference to a respawn function, this code allows the enemy to kill the player and immediately respawn the player at the most recent checkpoint

paulsinnett committed:
fix merge conflicts

paulsinnett committed:
Merge branch 'notleeandrew' of https://github.com/LSBUGPG/UnityLibrary into notleeandrew

paulsinnett committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into notleeandrew

# Conflicts:
#	ProjectSettings/ProjectSettings.asset

ClockworkAlexander commented:
* - Simple and efficient
* - Works without error
Wish - Camera clips through close meshes
